hasher-matcher-actioner/src/OpenMediaMatch/templates/components/banks/add_to_bank_form.html.j2

```diff
@@ -0,0 +1,37 @@
+{# Copyright (c) Meta Platforms, Inc. and affiliates. #}
+<button type="button" class="btn btn-light" data-bs-toggle="modal" data-bs-target="#{{bank_title}}">Add content</button>
```

Comment:
I wish this was an icon and not "Add content" but I didn't want to mess around with icon libraries either lol